diff --git a/node_modules/form-data/lib/browser.js b/node_modules/form-data/lib/browser.js
index 09e7c70..96076d6 100644
--- a/node_modules/form-data/lib/browser.js
+++ b/node_modules/form-data/lib/browser.js
@@ -1,2 +1,2 @@
 /* eslint-env browser */
-module.exports = typeof self == 'object' ? self.FormData : window.FormData;
+module.exports = typeof self == 'object' ? self.FormData : window? window.FormData: ()=>{};
