diff --git a/node_modules/react-native-elements/src/buttons/Button.js b/node_modules/react-native-elements/src/buttons/Button.js
index 1d8183f..7ecaa45 100644
--- a/node_modules/react-native-elements/src/buttons/Button.js
+++ b/node_modules/react-native-elements/src/buttons/Button.js
@@ -33,9 +33,7 @@ class Button extends Component {
   handleOnPress = () => {
     const { loading, onPress } = this.props;
 
-    if (!loading) {
       onPress();
-    }
   };
 
   render() {
diff --git a/node_modules/react-native-elements/src/config/colors.js b/node_modules/react-native-elements/src/config/colors.js
index b204851..c927b4d 100644
--- a/node_modules/react-native-elements/src/config/colors.js
+++ b/node_modules/react-native-elements/src/config/colors.js
@@ -37,5 +37,22 @@ export default {
       error: '#f44336',
       warning: '#ffeb3b',
     },
+    wx: {
+      primary: '#2196f3',
+      secondary: '#9C27B0',
+      grey: 'rgba(0, 0, 0, 0.54)',
+      success: '#4caf50',
+      error: '#f44336',
+      warning: '#ffeb3b',
+    },
+    web: {
+      primary: '#2196f3',
+      secondary: '#9C27B0',
+      grey: 'rgba(0, 0, 0, 0.54)',
+      success: '#4caf50',
+      error: '#f44336',
+      warning: '#ffeb3b',
+    },
+
   },
 };
diff --git a/node_modules/react-native-elements/src/config/colorsDark.js b/node_modules/react-native-elements/src/config/colorsDark.js
index bb32e8d..b766ee6 100644
--- a/node_modules/react-native-elements/src/config/colorsDark.js
+++ b/node_modules/react-native-elements/src/config/colorsDark.js
@@ -37,5 +37,21 @@ export default {
       error: '#bf2c24',
       warning: '#cfbe27',
     },
+    wx: {
+      primary: '#2196f3',
+      secondary: '#9C27B0',
+      grey: 'rgba(0, 0, 0, 0.54)',
+      success: '#4caf50',
+      error: '#f44336',
+      warning: '#ffeb3b',
+    },
+    web: {
+      primary: '#2196f3',
+      secondary: '#9C27B0',
+      grey: 'rgba(0, 0, 0, 0.54)',
+      success: '#4caf50',
+      error: '#f44336',
+      warning: '#ffeb3b',
+    },
   },
 };
