diff --git a/node_modules/react-native-easy-toast/index.js b/node_modules/react-native-easy-toast/index.js
index d30971e..750594a 100755
--- a/node_modules/react-native-easy-toast/index.js
+++ b/node_modules/react-native-easy-toast/index.js
@@ -40,7 +40,6 @@ export default class Toast extends Component {
     show(text, duration, callback, onPress) {
         this.duration = typeof duration === 'number' ? duration : DURATION.LENGTH_SHORT;
         this.callback = callback;
-        console.log(typeof onPress)
         if(typeof onPress === 'function')
             this.onPress = onPress
         this.setState({
@@ -108,11 +107,11 @@ export default class Toast extends Component {
                 pos = height - this.props.positionValue;
                 break;
         }
-
+        const {containerStyle= {}} = this.props
         const view = this.state.isShow ?
         <TouchableWithoutFeedback onPress={this.onPress} >
             <View
-                style={[styles.container, { top: pos }]}
+                style={[styles.container, { top: pos }, containerStyle]}
                 pointerEvents="auto"
             >
                 <Animated.View
